ssl/ssl_key_share.cc

```diff
@@ -303,12 +531,21 @@ CONSTEXPR_ARRAY NamedGroup kNamedGroups[] = {
     {NID_CECPQ2, SSL_CURVE_CECPQ2, "CECPQ2", "CECPQ2"},
 };
 
+CONSTEXPR_ARRAY uint16_t kPQGroups[] = {
+    SSL_CURVE_CECPQ2,
```

Comment:
the KEM API doesn't support `CECPQ2`.

Reply:
No, but this is a collection of all PQ groups, regardless of whether the KEM APIs support them. This list is not used to determine if a KEM API should be called; its used for other handshake logic:

```
// is_post_quantum_group returns true if |id| is a group ID associated
// with a post-quantum group (either hybrid or standalone).
// Otherwise, it returns false.
static bool is_post_quantum_group(uint16_t id) {
  for (const uint16_t pq_group_id : PQGroups()) {
    if (id == pq_group_id) {
      return true;
    }
  }
  return false;
}
```

https://github.com/aws/aws-lc/blob/dbb1ea4499804e046276a1d1ab61187e142d3ff4/ssl/extensions.cc#L343-L346

https://github.com/aws/aws-lc/blob/dbb1ea4499804e046276a1d1ab61187e142d3ff4/ssl/extensions.cc#L408-L410

https://github.com/aws/aws-lc/blob/dbb1ea4499804e046276a1d1ab61187e142d3ff4/ssl/extensions.cc#L2303-L2311

https://github.com/aws/aws-lc/blob/dbb1ea4499804e046276a1d1ab61187e142d3ff4/ssl/extensions.cc#L2549-L2553